### ISSUE TITLE
新增功能：Gradient Boosting Strategy（梯度提升策略）

### ISSUE THREAD

**Issue by Kaoruha** [Kaoruha/GinkGO#2540]
## 功能描述
**XGBoost**： XGBoost策略实现。

**LightGBM**： LightGBM实现。

**特征重要性**：特征重要性分析。

**模型集成**：多模型集成。

## 研究参考资料
- Gradient Boosting
- XGBoost
- LightGBM

## 实现复杂度
中等 - 需要ML框架。

## 与Ginkgo的契合度
- 策略模块
- GBDT
- ML
